src/components/Explore/SingleLocationChart.tsx

```diff
@@ -113,12 +158,25 @@ const SingleLocationChart: React.FC<{
   marginRight = 20,
   barOpacity,
   barOpacityHover,
+  dateRange,
+  yTickFormat,
+  yTooltipFormat,
+  xTickTimeUnit,
 }) => {
-  const dateFrom = new Date('2020-03-01');
-  const today = new Date();
-  const dateTo = today;
+  const [dateFrom, dateTo] = dateRange;
+
+  const inDateRange = (date: Date): boolean =>
+    dateFrom <= date && date <= dateTo;
+
+  const maxPerSeries = seriesList.map(series => {
+    const dataInRange = series.data.filter(d => inDateRange(new Date(d.x)));
+    const seriesMax = max(dataInRange.map(getY)) || 1;
+    return seriesMax;
+  });
```

Comment:
How about moving this code to `Explore/utils`?

```ts
function inDateRange(date: Date, dateFrom:Date , dateTo:Date) {
  return dateFrom <= date && date <= dateTo;
}

function maxPerSeries(
  seriesList: Series[], 
  dateFrom: Date, 
  dateTo: Date
  getY: (d:Column) => number, 
) {
  const dataInRange = series.data.filter(d => inDateRange(new Date(d.x), dateFrom, dateTo));
  return max(dataInRange.map(getY)) || 1;
}
```

The second function is a bit annoying though, too many arguments. Your call @chasulin !

Reply:
[updated/moved to utils](https://github.com/covid-projections/covid-projections/pull/3670/commits/54922c7deb35599ae716c2399c9ad3dcf9c61027)